src/Sextant.XamForms/Mixins/DependencyResolverMixins.cs

```diff
@@ -0,0 +1,107 @@
+            return dependencyResolver;
+        }
+
+        /// <summary>
+        /// Initializes sextant.
+        /// </summary>
+        /// <param name="dependencyResolver">The dependency resolver.</param>
+        /// <param name="mainScheudler">The main scheudler.</param>
+        /// <param name="backgroundScheduler">The background scheduler.</param>
+        /// <returns>The dependencyResovler.</returns>
+        public static IMutableDependencyResolver RegisterNavigationView(this IMutableDependencyResolver dependencyResolver, IScheduler mainScheudler, IScheduler backgroundScheduler)
+        {
+            var vLocator = Locator.Current.GetService<IViewLocator>();
+
+            dependencyResolver.RegisterLazySingleton(() => new NavigationView(mainScheudler, backgroundScheduler, vLocator), typeof(IView), NavigationView);
+            return dependencyResolver;
+        }
+
+        /// <summary>
+        /// Registers the view stack service.
+        /// </summary>
+        /// <typeparam name="T">The view stack service type.</typeparam>
+        /// <param name="dependencyResolver">The dependency resolver.</param>
+        /// <returns>The dependencyResovler.</returns>
+        public static IMutableDependencyResolver RegisterViewStackService(this IMutableDependencyResolver dependencyResolver)
+        {
+            var view = Locator.Current.GetService<IView>(NavigationView);
+            dependencyResolver.Register<IViewStackService>(() => new ViewStackService(view));
+            return dependencyResolver;
+        }
+
+        /// <summary>
+        /// Registers the view stack service.
+        /// </summary>
+        /// <typeparam name="T">The view stack service type.</typeparam>
+        /// <param name="dependencyResolver">The dependency resolver.</param>
+        /// <param name="factory">The factory.</param>
+        /// <returns>The dependencyResovler.</returns>
+        public static IMutableDependencyResolver RegisterViewStackService<T>(this IMutableDependencyResolver dependencyResolver, Func<IView, T> factory)
+            where T : IViewStackService
+        {
+            var view = Locator.Current.GetService<IView>(NavigationView);
+            dependencyResolver.RegisterLazySingleton<T>(() => factory(view));
+            return dependencyResolver;
+        }
+
+        /// <summary>
+        /// Registers the specified view with the Splat locator.
+        /// </summary>
+        /// <typeparam name="TView">The type of the view.</typeparam>
+        /// <typeparam name="TViewModel">The type of the view model.</typeparam>
+        /// <param name="dependencyResolver">The dependency resolver.</param>
+        /// <param name="contract">The contract.</param>
+        /// <returns>The dependencyResovler.</returns>
+        public static IMutableDependencyResolver RegisterView<TView, TViewModel>(this IMutableDependencyResolver dependencyResolver, string contract = null)
```

Comment:
I would add a `Func<IViewFor<TViewModel>>` version of this in case the user has a odd constructor